examples/serre_green_naghdi_1d/serre_green_naghdi_dingemans.jl

```diff
@@ -0,0 +1,49 @@
+using OrdinaryDiffEq
+using DispersiveShallowWater
+using SummationByPartsOperators: upwind_operators, periodic_derivative_operator
+
+###############################################################################
+# Semidiscretization of the Svärd-Kalisch equations
+
+bathymetry_type = bathymetry_variable # or bathymetry_mild_slope
+equations = SerreGreenNaghdiEquations1D(bathymetry_type;
+                                        gravity_constant = 9.81)
+
+initial_condition = initial_condition_dingemans
+boundary_conditions = boundary_condition_periodic
+
+# create homogeneous mesh
+coordinates_min = -140.0
+coordinates_max = 100.0
```

Comment:
These are different to the Dingemans elixirs of the other equations. I see you used these values in your preprint. Do you think it makes sense take the same interval bounds across all Dingemans elixirs for better comparability (either use $[-138, 46]$ everywhere or $[-140, 100]$ everywhere)?

Reply:
I would leave that up to you - which version would you like to have?

Reply:
To be able say which version I prefer it would be interesting to know why you used other values in the first place? To avoid wave interaction between the waves entering the periodic domain in the left and some small waves, which still existed in the left area from the initial wave train?

Reply:
Yes, exactly

Reply:
Did you observe any influence of these interactions? At least visually it looks like the waves from the initial wave train are really small also in the smaller domain. And also in the larger domain you have waves entering the left again. I think if the influence of the interactions are not too big I would keep the smaller domain to have a better resolution.

Reply:
Done in a0cdae4